Sage/carbon
# refactor(help): refactor files to typescript FE-4796

### Proposed behaviour


Refactors `Help` to be TypeScript

### Current behaviour


`Help` is JavaScript

### Checklist




#### QA


### Additional context



### Testing instructions


Regression test in storybook and use linked sandbox

The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.


https://codesandbox.io/s/jovial-cherry-z0uek6?file=/src/index.tsx -- export locally
